api/v1alpha1/storagesystem_types.go

```diff
@@ -30,8 +30,14 @@ import (
 type StorageKind string
 
 const (
-	// ConditionResourcePresent communicates the status of underlying resource
-	ConditionResourcePresent conditionsv1.ConditionType = "ResourcePresent"
+	//ConditionStorageSystemInValid communicates if storagesystem CR is inValid
+	ConditionStorageSystemInValid conditionsv1.ConditionType = "StorageSystemInValid"
```

Comment:
Here and in every other instance: the v in invalid should never be capitalized, it's one word and not two. Look for both `InValid` and `inValid`.

Reply:
Done